demo/app/components/keyboard-control-layer/keyboard-control-layer.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { KeyboardControlService } from '../../../../src/services/keyboard-control/keyboard-control.service';
+import { KeyboardAction } from '../../../../src/models/ac-keyboard-action.enum';
+
+@Component({
+  selector: 'keyboard-control-layer',
+  template: '',
+})
+export class KeyboardControlLayerComponent implements OnInit, OnDestroy {
+  constructor(private keyboardControlService: KeyboardControlService) {
+
+  }
+
+  ngOnInit() {
+    this.keyboardControlService.setKeyboardControls({
+      W: { action: KeyboardAction.CAMERA_FORWARD },
+      S: { action: KeyboardAction.CAMERA_BACKWARD },
+      D: { action: KeyboardAction.CAMERA_RIGHT },
+      A: { action: KeyboardAction.CAMERA_LEFT },
+    }, (keyEvent: KeyboardEvent) => {
```

Comment:
Arrow key also has a string value, and you can get it using `String.fromCharCode` and use it as key. for example:
```
[String.fromCharCode(37)]: { action: KeyboardAction.CAMERA_LEFT },
```

Or, you can use `keyMappingFn`, and return any string you want using the `KeyboardEvent` that exposed. take a look at the second parameter of the register function.